### DISCUSSION TITLE
posts/Go/install/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#287] [Announcements]
https://www.liwenzhou.com/posts/Go/install/ 

 李文周的Blog提供免费的全套Go语言学习教程，本文详细介绍了Go语言的安装步骤，还介绍了GOPATH和GOROOT是什么，以及如何配置GOPROXY代理。使用VSCode作为Go语言开发工具需要如何配置，介绍了如何下载vscode插件来满足开发需求。然后我们亲自动手编写了第一个Go语言程序，并演示了如何使用go build命令编译程序并在终端中执行程序，最后介绍了Go语言中如何进行跨平台编译。

**Comment by pro911** [comment]
Windows 下根据设置交叉编译mac包 报错
